source/test/java/com/restfb/types/PostTest.java

```diff
@@ -294,4 +297,30 @@ public void checkV2_8_feedTargetingScheduleUntil() {
     Date ts = examplePost.getFeedTargeting().getRelevantUntilTs();
     assertEquals(1457620560000L, ts.getTime());
   }
+
+  @Test
+  public void checkV2_5_feedTargetingWithRegions() {
+    Post examplePost = createJsonMapper().toJavaObject(jsonFromClasspath("v2_5/post-with-feedtargeting-regions"), Post.class);
```

Comment:
Here are a couple of screenshots showing the v2.5 and v2.8 request of the same object... 
<img width="577" alt="screen shot 2016-10-12 at 17 26 19" src="https://cloud.githubusercontent.com/assets/719204/19329970/2746a4ba-90a1-11e6-8c75-6ff6927b9ac6.png">
<img width="641" alt="screen shot 2016-10-12 at 17 26 41" src="https://cloud.githubusercontent.com/assets/719204/19329971/274ad210-90a1-11e6-9044-a65cafbea3eb.png">

The breaking thing is that the "regions" in 2.5 is a map, but in 2.8 is a list.